modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install_{context}"]
+= Configuring multiple vCenters during installation
+
+To configure multiple vCenters during installation:
+
```

Comment:
Do we need to add wording that - multiple vcenters can't be added as day-2 operation?

Reply:
@gnufied I've added more explicit information about this in the Over section that is just above the procedure. Let me know if that is sufficient.